packages/schematics/src/lib-versions.ts

```diff
@@ -7,7 +7,7 @@ export const ngrxStoreFreezeVersion = '0.2.4';
 export const nxVersion = '*';
 export const schematicsVersion = '*';
 export const latestMigration = '20180507-create-nx-json';
-export const prettierVersion = '1.13.7';
+export const prettierVersion = '1.15.1';
```

Comment:
Prettier's already at 1.15.2 FYI

Reply:
Awesome, looks like there was a lot of bug fixes pertaining to html formatting actually. :) glad we got those in.